Classification/Experiments/src/main/java/org/tribuo/classification/experiments/Test.java

```diff
@@ -134,33 +126,12 @@ public static Pair<Model<Label>,Dataset<Label>> load(ConfigurableTestOptions o)
         Path modelPath = o.modelPath;
         Path datasetPath = o.testingPath;
         logger.info(String.format("Loading model from %s", modelPath));
-        Model<?> tmpModel;
-        if (o.protobufModel) {
-            tmpModel = Model.deserializeFromFile(modelPath);
-        } else {
-            try (ObjectInputStream mois = new ObjectInputStream(new BufferedInputStream(new FileInputStream(modelPath.toFile())))) {
-                tmpModel = (Model<?>) mois.readObject();
-            } catch (ClassNotFoundException e) {
-                throw new IllegalArgumentException("Unknown class in serialised model", e);
-            }
-        }
+        Model<?> tmpModel = Model.deserializeFromFile(modelPath);
         Model<Label> model = tmpModel.castModel(Label.class);
         logger.info(String.format("Loading data from %s", datasetPath));
         Dataset<Label> test;
         switch (o.inputFormat) {
             case SERIALIZED:
```

Comment:
Isn't this kind of a breaking change? Code that previously expected java serialization would not inadvertently be using a proto serialization path.